storage/store.go

```diff
@@ -1007,8 +1045,19 @@ func (s *Store) GroupStorage(groupID uint64) multiraft.WriteableGroupStorage {
 	defer s.mu.RUnlock()
 	r, ok := s.ranges[int64(groupID)]
 	if !ok {
-		log.Warningf("%p requested nonexistent range with raft ID %d", s, groupID)
-		return nil
+		var err error
+		r, err = NewRange(&proto.RangeDescriptor{
+			RaftID: int64(groupID),
```

Comment:
I agree; that's why the change hasn't happened yet (we could unilaterally switch to unsigned to match etcd/raft, but if we want to change to signed we have to convince them to change too).